tests/unittest/auto_deploy/singlegpu/custom_ops/mla/test_trtllm_mla_op.py

```
@@ -0,0 +1,1514 @@
+    # Production MLA uses kv_factor=1 (K and V are the same latent data).
+    kv_cache = torch.zeros(
+        max_num_pages,
+        1,
+        num_kv_heads,
+        page_size,
+        latent_dim,
+        dtype=dtype,
+        device=device,
+    )
+
+    kv_lengths = [pos + slen for pos, slen in zip(input_positions, seq_lengths)]
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)